cinema/tests/test_movie_view_set.py

```diff
@@ -0,0 +1,130 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from cinema.models import Movie, Genre, Actor
+from django.contrib.auth import get_user_model
+from rest_framework.authtoken.models import Token
+from io import BytesIO
+from PIL import Image
+
+User = get_user_model()
+
+def create_genre(name: str) -> Genre:
+    return Genre.objects.create(name=name)
+
+def create_actor(first_name: str, last_name: str) -> Actor:
+    return Actor.objects.create(first_name=first_name, last_name=last_name)
+
+def create_movie(title: str, genres=None, actors=None) -> Movie:
+    movie = Movie.objects.create(title=title, description="Sample description", duration=120)
+    if genres:
+        movie.genres.set(genres)
+    if actors:
+        movie.actors.set(actors)
+    return movie
+
+def generate_image_file():
```

Comment:
This is a code quality issue. The function generate_image_file doesn't have a docstring. It's a good practice to write docstrings for all functions, methods and classes. The docstring should explain what the function does, its parameters, its return value, and any exceptions that it raises.